test/index.js

```diff
@@ -44,7 +65,12 @@ describe('Public API', () => {
 
     describe('addEntry', () => {
 
+        it('should not be callable before the runtime environment has been configured', () => {
+            expect(() => api.addEntry('entry', 'main.js')).to.throw();
+        });
```

Comment:
Exactly what I thought after copy/pasting that part for every method of the API.

I wanted a 2nd opinion before removing everything :)